cleaners/vuze.xml

```diff
@@ -18,14 +18,16 @@
     You should have received a copy of the GNU General Public License
     along with this program. If not, see <http://www.gnu.org/licenses/>.
 
+    @app Vuze
     @url http://www.vuze.com/
+    @os Windows, Linux
     @cleanerversion v2.0.0
-    @cleanerdate 2019-03-11
+    @cleanerdate 2019-03-13
     @cleanerby https://github.com/Tobias-B-Besemer
```

Comment:
I created this in 2009, but the credit just mentions you. 

😢

https://github.com/bleachbit/bleachbit/commit/efb1b7de9409a59bae7dba40db6f57f0557ef545#diff-e3a08ce16efae5e4e6cbfcef163ffe87

Reply:
Sorry, I c&p the header! Seems I just forgot to add "??? & " before (like I did on other cleaners)! I add always "??? & " because I don't know who already worked on them...
I will add you! Someone else, too?

Reply:
Is there any chance to browse the history of a single file on git to see who have made PR to this file ???

Reply:
I guess I found it... https://github.com/bleachbit/bleachbit/commits/master/cleaners/vuze.xml

Reply:
Yes, the Git history is helpful. Caveat: Older change records were imported from SVN when the authors were written in the commit body instead of the commit header.

Reply:
I miss the old COBOL times when programmers got payed per line the written... ;-) :-D